csharp/src/Ice/Endpoint.cs

```diff
@@ -281,5 +281,33 @@ internal static class EndpointExtensions
             }
             return sb;
         }
+
+        internal static StringBuilder AppendEndpointList(
+            this StringBuilder sb,
+            IReadOnlyList<Endpoint> endpoints)
+        {
+            if (endpoints[0].Protocol == Protocol.Ice1)
```

Comment:
Should we add a `Debug.Assert(endpoints.Count > 0)`? Or just return an empty string?

Reply:
Added assert,